src/assets/data/testimonial/index.js

```diff
@@ -10,16 +10,16 @@ const data = {
   testimonials: [
     {
       quote:
-        'This app is on one assistant statement caught agency, the atprior from peace counter. A to is home, between history; With the headline is an the narrow size part. In ride ourselves,you to out',
-      author: 'Jhon Smith',
-      dsignation: 'Designer',
+        "Working with service mesh technology, I find one of the most useful tools is Meshery! I use Meshery to run POCs for our customers to help them adopt, and then, ongoing to check our mesh configuration. The most fantastic part about Meshery is it's performance management features.",
+      author: 'Mohsen Houshmand',
+      dsignation: 'Intraserve Systems',
       thumb: image1
     },
     {
         quote:
-          'This app is on one assistant statement caught agency, the atprior from peace counter. A to is home, between history; With the headline is an the narrow size part. In ride ourselves,you to out',
-        author: 'Smith Doe',
-        dsignation: 'Marketer',
+          'Our Complex, Dynamic, and Intelligent system demands leverage different tools to efficiently manage service exchange and ecosystem at Global Scale; for instance, we need to identify right fit Service Mesh to address varying workload scenarios, distributed systems across locations, different security tiers, minimal latency etc. Meshery helps us curate Service Mesh for tailored need by helping us running simulations and identifying the right fit Service Mesh that works efficiently across diverse ecosystem of tools.',
+        author: 'Cloudless',

```

Comment:
Is the author supposed to be Cloudless? @leecalcote

Reply:
@Tanuj22 Good question. No, it’s not. Cloudless is a division of HPE Labs. 

For now, we can use “Distinguished Technologist” as the author.